editor/src/components/canvas/canvas-strategies/absolute-move-strategy.tsx

```diff
@@ -110,7 +111,7 @@ export const absoluteMoveStrategy: CanvasStrategy = {
       )
     }
     // Fallback for when the checks above are not satisfied.
-    return emptyStrategyApplicationResult
+    return failedStrategyApplicationResult
```

Comment:
Hmmm this raises an interesting semantical point here, because I would interpret a "failed" result to imply that we tried to apply it but something went wrong. That makes sense in the re-parenting strategies when the target doesn't accept children. But in this case we haven't tried to apply the strategy as such, because there has been no drag yet. If we mark this as failed (and grey out the text in the picker) I think that risks giving the user the impression that they can't apply this strategy. Maybe it would be better for this to return an empty successful result?

Reply:
oooooh that's a very nice catch! would this show the failed state for all strategies before they reach the drag threshold? that's definitely a mistake!

Reply:
fixed in b62e7b05842dadd58dca881eba7c34d100649e48